package.json

```diff
@@ -44,6 +44,9 @@
     "html-webpack-plugin": "^4.2.0",
     "minimist": "^1.2.5",
     "node-sass": "^4.14.0",
+    "npm": "^6.14.8",
```

Comment:
Remove dependency `npm` and `optimize-css-assets-webpack-plugin` (We removed both, so if you need them for things to build you may have to merge back develop)